playwright.base.config.js

```diff
@@ -8,6 +8,29 @@ const getEnvNum = (envVar, defaultValue) => {
   return defaultValue;
 };
 
+const getEnvString = (envVar, defaultValue) => {
+  if (envVar?.trim().length) {
+    return envVar;
+  }
+
+  return defaultValue;
+};
+
+const getEnvBool = (envVar, defaultValue) => {
+  if (!envVar) {
+    return defaultValue;
+  }
+  const val = envVar.toLowerCase();
+  if (['true', 't', '1', 'on'].includes(val)) {
```

Comment:
what fresh hell is this, `t` `on` === true?

Reply:
You're right, I should use the [fresh hell that is already](https://github.com/strapi/strapi/blob/15a78717c0e0d632c7c1f4bce69107de2ccc95d4/packages/core/utils/src/parse-type.ts#L117) exported from strapi/utils